frontend/src/components/table/Table.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from "react";
+import Card from "../card/Card";
+import classes from "./Table.module.scss";
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+
+type TableProps = {
+  columns: any[];
+  data: any[];
+};
+
+const Table: FunctionComponent<TableProps> = ({ columns, data }) => {
+  const table = useReactTable({
+    data,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+  });
+
+  return (
+    <div className={classes.root}>
```

Comment:
I noticed that `root` class isn't present in `Table.module.scss`. Do we need to also specify some styled for the root element or can we safely remove this className property?